weekly-updates/weekly-update.2015-06-19.md

```diff
@@ -0,0 +1,47 @@
+# io.js News
```

Comment:
totally up to this group, but I'd do it as a blanket thing, even posting as iojs on medium, maybe just title it "io.js and Node.js news"